src/shared/components/CategoryPicker/CategoryPicker.style.ts

```diff
@@ -4,8 +4,20 @@ import ToggleButton from '../ToggleButton'
 import { sizes, colors } from '@/shared/theme'
 
 export const Container = styled.div`
-  display: flex;
-  flex-wrap: wrap;
+  display: -webkit-box;
+  white-space: nowrap;
+  overflow: scroll;
+  ::-webkit-scrollbar {
+    display: none;
+  }
+  > * {
```

Comment:
I think those should be living outside of the component itself, in `StyledCategoryPicker` in `BrowseView.styles`. Without any additional styling the component should work as part of the layout, only the Browse View makes it span the full screen width